extensions/markdown-language-features/src/languageFeatures/copyFiles/shared.ts

```diff
@@ -70,21 +83,30 @@ export async function getMarkdownLink(document: vscode.TextDocument, ranges: rea
 	const edits: vscode.SnippetTextEdit[] = [];
 	let placeHolderValue: number = ranges.length;
 	let label: string = '';
-	let smartPaste: boolean = false;
+	let smartPaste = { paste: false, title: false };
```

Comment:
For documentation, it may be worth either adding an interface or class that you can document what `paste` / `title` mean